satellitevu/apis/otm.py

```diff
@@ -22,20 +24,22 @@ def post_feasibility(
         coordinates: Union[Tuple[float, float], Tuple[float, float, float]],
         date_from: datetime,
         date_to: datetime,
-        day_night_mode: str = "day-night",
+        day_night_mode: Literal["day", "night", "day-night"] = "day-night",
         **kwargs,
     ):
         """
         Creates a tasking feasibility request.
 
         Args:
-            coordinates: An array of coordinates.
+            coordinates: An array of coordinates - (longitude, latitude) or
+            (longitude, latitude, altitude).
 
             date_from: datetime representing the start date of the feasibility request.
 
             date_to: datetime representing the end date of the feasibility request.
 
-            day_night_mode: The mode of data capture, defaults to "day-night".
+            day_night_mode: String representing the mode of data capture. Allowed values
+            include ["day", "night", "day-night"]. Defaults to "day-night".
```

Comment:
"Allowed values *include*" suggests to me there are more. It's just "Allowed values *are*", right?

Reply:
makes sense to me!